custom_model_runner/datarobot_drum/drum/gpu_predictors/base.py

```diff
@@ -139,6 +139,7 @@ def configure(self, params):
             verify_ssl=self.verify_ssl,
             total_deployment_stages=self.num_deployment_stages,
         )
+        self.python_model_adapter.load_custom_hooks()
```

Comment:
We already have this at line 133; why add this again (with no if check)?

Reply:
oh, correct. I haven't seen that line, but I had an impression that it was not working at all... need to check.

Reply:
removed.